network-api/networkapi/wagtailpages/donation_modal.py

```diff
@@ -35,6 +38,25 @@ class DonationModal(TranslatableMixin, models.Model):
         default="Yes, I'll chip in",
     )
 
+    donate_url = models.CharField(
+        max_length=255,
+        default="?form=donate",
```

Comment:
Now making the donate_url field required, with the default of `?form=donate`.

Note: since this is a new field, no custom migration is required!